### PR TITLE
Tech Debt & Aug Dep Bumps

### PR DESCRIPTION
**Pull request by stdavis**
- prettier
- Travis -> GitHub Actions
- CRA -> Vite

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,3 @@
-REACT_APP_DISCOVER=chance-source-liquid-nelson
-REACT_APP_VERSION=$npm_package_version
-REACT_APP_ENVIRONMENT=prod
+VITE_DISCOVER=chance-source-liquid-nelson
+VITE_VERSION=$npm_package_version
```
Comment (steveoh) [inline .env:2]: I don't think this works? Does it?

File: .env [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+VITE_ENVIRONMENT=prod
```
Comment (steveoh) [inline .env:3]: I think there is a built in for this called mode

File: index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (steveoh) [inline index.html:2]: Did this need to change when it's Spanish?

File: src/Helpers.js [hunk 4]
```diff
@@ -1,4 +1,3 @@
-
 const toQueryString = (obj) => {
```
Comment (steveoh) [inline src/Helpers.js:1]: This can be replace with utilities

File: src/components/LayerSelector/LayerSelector.css [hunk 5]
```diff
@@ -28,10 +28,10 @@
   margin-top: 0px;
   margin-bottom: 0px;
 }
-.layer-selector-item.radio input[type=checkbox],
-.layer-selector-item.checkbox input[type=checkbox],
-.layer-selector-item.radio input[type=radio],
-.layer-selector-item.checkbox input[type=radio] {
+.layer-selector-item.radio input[type='checkbox'],
```
Comment (steveoh) [inline src/components/LayerSelector/LayerSelector.css:31]: Can this be replaced by the widgets?

File: src/hooks.js [hunk 6]
```diff
@@ -1,5 +1,4 @@
-import { useState, useEffect } from 'react';
-
+import { useEffect, useState } from 'react';
```
Comment (steveoh) [inline src/hooks.js:1]: We should move this to utilities

**Review by steveoh — APPROVED** [review]
Great job!

File: .env (hunk 1 above)
Comment (stdavis) [reply .env:2]: You're correct. It doesn't work. I implemented a different solution.

File: .env (hunk 2 above)
Comment (stdavis) [reply .env:3]: That's correct and I wasn't using it anyways.

File: index.html (hunk 3 above)
Comment (stdavis) [reply index.html:2]: It probably should but I haven't noticed it causing any issues in my testing or in production. I'm not sure that it makes a difference if it's switched dynamically on the client.

File: src/components/LayerSelector/LayerSelector.css (hunk 5 above)
Comment (stdavis) [reply src/components/LayerSelector/LayerSelector.css:31]: Yes, at some point #123

File: src/Helpers.js (hunk 4 above)
Comment (stdavis) [reply src/Helpers.js:1]: #123

File: src/hooks.js (hunk 6 above)
Comment (stdavis) [reply src/hooks.js:1]: #123

**Pull request merged**